### PR TITLE
Feature: ElasticSearch GUI Clients

### PR DESCRIPTION
**Pull request by llaville**
Hello @cytopia 

Here is a second proposal to add GUI ElasticSeach Clients to enlarge already ELK Stack available in Devilbox 1.0

### CODE REVIEW

File: flames/elastic/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+| Container host name   | `mirage`                           | 
+| Container name        | `mirage`                           | 
+| Mount points          | none                               | 
+| Exposed port          | `3030` (can be changed via `.env`) |  
+| Available at          | `http://localhost:3030`            | 
+
+
+### CORS settings
+
+To make sure you have enabled CORS settings for your ElasticSearch instance, add the following lines
```
Comment (cytopia) [inline flames/elastic/README.md:52]: Why is this required?

File: flames/elastic/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+| Available at          | `http://localhost:3030`            | 
+
+
+### CORS settings
+
+To make sure you have enabled CORS settings for your ElasticSearch instance, add the following lines 
+in the elastic container, environment section:
+
+```yaml
+    environment:
```
Comment (cytopia) [inline flames/elastic/README.md:56]: I struggled a bit and it took some time to understand, that this needs to go into the docker-compose.override.yml for ELK stack,

What is the benefit of adding it and what happens if you don't? If it is crucial and doesn't interfere with other functionality I can already add it into it, so that there's no need for extra configuration.

**Review by cytopia — APPROVED** [review]
Looks solid :+1:

File: flames/elastic/README.md (hunk 1 above)
Comment (llaville) [reply flames/elastic/README.md:52]: Because you cannot connect to Mirage or DejaVu clients:  see https://github.com/appbaseio/mirage/issues/13

File: flames/elastic/README.md (hunk 2 above)
Comment (llaville) [reply flames/elastic/README.md:56]: CORS setting is mandatory; See Mirage issue https://github.com/appbaseio/mirage/issues/13
